tests/roots/test-docstrings/dummy_django_app/models.py

```diff
@@ -2,7 +2,8 @@
 from django.contrib.contenttypes.models import ContentType
 from django.db import models
 
-from sphinxcontrib_django.docstrings.config import CHOICES_LIMIT
+#: Default value of ``django_choices_to_show``
+CHOICES_LIMIT = 10
```

Comment:
Now we have the value `10 hardcoded at two places: The config definition and the tests. I think this is not ideal, since if we change the e.g. the new default value of the config option without adjusting the tests, we might loose test coverage.
So I suggest to keep the centralized definition of the default choices limit and importing it here.

Reply:
@timoludwig I am not sure, that I fully understand you idea. 

You want to keep `CHOICES_LIMIT=10` in `from sphinxcontrib_django.docstrings.config` and use it as default in `app.add_config_value("django_choices_to_show", 10, True)` line in `__init__.py`?

If so, I am ok with it and will add changes. Otherwise we will have not fully testable case with limit change.

Reply:
Resolved in 3324733882357121fed20668495e49a82819ba90